setup.py

```diff
@@ -15,22 +16,31 @@
 
 PATH_ROOT = os.path.dirname(__file__)
 builtins.__LIGHTNING_BOLT_SETUP__ = True
+builtins.__LIGHTNING_SETUP__ = True
+
+import sys
+
+# remove local path to force loading global models
+sys.path.remove('')
+import pytorch_lightning
+
+# read init content
+with open(os.path.join(pytorch_lightning.__path__[0], '__init__.py'), 'r') as f:
+    init_content = f.read()
 
-import pytorch_lightning_bolts  # noqa: E402
+# delete previously loaded lightning
+del pytorch_lightning
+sys.modules.pop('pytorch_lightning', None)
 
+# re-add locals to path
+sys.path = [''] + sys.path
 
-def load_requirements(path_dir=PATH_ROOT, comment_char='#'):
-    with open(os.path.join(path_dir, 'requirements.txt'), 'r') as file:
-        lines = [ln.strip() for ln in file.readlines()]
-    reqs = []
-    for ln in lines:
-        # filer all comments
-        if comment_char in ln:
-            ln = ln[:ln.index(comment_char)]
-        if ln:  # if requirement is not empty
-            reqs.append(ln)
-    return reqs
+# overwrite init
+with open(os.path.join(os.path.abspath(PATH_ROOT),
+                       'pytorch_lightning', '__init__.py'), 'w') as f:
+    f.write(init_content)
```

Comment:
this assumes the lightning is already installed...
what about the case user wants to install all at once :]

Reply:
good point. although i argue bolts needs lightning as a dependency?

Reply:
Yes, but when users just install bolts, lightning comes, but the init-replacement mechanism does not work, since it is installed at the same time and not already installed @williamFalcon